test/controllers/form.controller.test.ts

```diff
@@ -0,0 +1,54 @@
+import 'reflect-metadata'
+
+import { assert } from 'chai'
+import { Application } from 'express'
+import { BAD_REQUEST, MOVED_TEMPORARILY } from 'http-status-codes'
+import { deepEqual, instance, mock, when } from 'ts-mockito'
+import { createApp } from '../application.factory'
+
+import 'app/controllers/form.controller'
+import ValidationErrors from 'app/models/validationErrors'
+import { FORM_PAGE_URI } from 'app/paths'
+import formSchema from 'app/schemas/form.schema'
+import FormValidator from 'app/utils/formValidator.util'
+
+import { setupSuperTest } from 'test/supertest.factory'
+
+
+describe('FormController', () => {
+  it('should redirect successfully if validator returns no errors', async () => {
```

Comment:
Can we add 2 inner describe blocks here:
1. GET - ensure that page loads correctly
2. POST - ensure form submission is handled correctly